yarl/_url.py

```diff
@@ -1112,6 +1112,19 @@ def update_query(self, *args: Any, **kwargs: Any) -> "URL":
             self._val._replace(query=self._get_str_query(query) or ""), encoded=True
         )
 
+    def without_query_params(self, *query_params: str) -> "URL":
+        """Remove some keys from query part and return new URL."""
+        params_to_remove = set(query_params) & set(self.query.keys())
```

Comment:
```suggestion
        params_to_remove = set(query_params) & self.query.keys()
```

Does this need to be wrapped in a set?

Reply:
Thanks, I didn't know that `set`'s `&` operator accepts `_KeysView` on the right-hand-side.